api/controller/controller.go

```diff
@@ -239,6 +239,18 @@ func (c *Client) GetControllerAccess(user string) (permission.Access, error) {
 	return permission.Access(results.Results[0].Result.Access), nil
 }
 
+// ConfigSet updates the passed controller configuration values. Any
+// settings that aren't passed will be left with their previous
+// values.
+func (c *Client) ConfigSet(values map[string]interface{}) error {
+	if c.BestAPIVersion() < 5 {
+		return errors.Errorf("controller must be version 2.3.3 or higher to update controller config")
```

Comment:
in general we have avoided mentioning specific versions
"this version of juju does not support updating controller config values" or something

Reply:
Ok, I'll change that.